templates/todo/task_list.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>TODO list</h1>
+    <button style="float: right; bottom: 50px; position: relative" class="btn btn-primary"
+            type="submit"
+            value="submit">
+        <a role="button" style="color: white"
+           href="{% url 'todo:task-create' %}">Add task</a>
+    </button>
+    <br>
+
+    {% if task_list %}
+        {% for task in task_list %}
+            <br>
+            <div class="form-group">
+                <b>{{ task.content }}</b>
+
+
+                {% if task.is_done %}
+                    <b><span class="done">Done</span></b>
+                    <form method="post" action="{% url 'todo:task_change_status' pk=task.id %}">
+                        {% csrf_token %}
+
+                        <button style="float: right" type="submit" class="btn btn-blurred">
```

Comment:
Add class btn-right here and remove styles here and in line 36. Add this to styles.css:

.btn-right{
float: right;
}

```suggestion
                        <button type="submit" class="btn btn-blurred btn-right">
```